src/clj/doit/todo/route.clj

```diff
@@ -2,7 +2,7 @@
   (:require [doit.todo.handler :as handler]
             [doit.middleware :as mw]))
 
-(def routes {"" {:post     (mw/wrap-token handler/create-todo)
-                 :get      (mw/wrap-token handler/list-todos)
-                 [:id "/"] {:put    (mw/wrap-token handler/update-todo)
-                            :delete (mw/wrap-token handler/delete-todo)}}})
+(def routes {"" {:post     (mw/wrap-token handler/create!)
```

Comment:
It'd be nice if you could alias `doit.todo.handler` to `todo` instead. This would make handlers more explicit and easy to understand (ex: `todo/create!`)